frontend/src/worksheets/FilterWorkSheets.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+function FilterWorkSheets(props) {
+  const { onStatusChange, status } = props
+
+  const handleInputChange = (event) => {
+    onStatusChange(event.target.value)
+  }
+
+  return (
+    <div className="my-3">
+      <form>
+        <select
+          value={status}
+          className="btn btn-success"
+          onChange={handleInputChange}
+          name="filterWorkSheetByStatus"
+        >
+          <option value="CREATED">Létrehozott</option>
```

Comment:
Akár itt használhatnánk a TranslationForWorkSheet konstantsait, ha változik a status terminológiája, akkor egy helyen kell így átírni. Akár oda lehet egy új állapotot is felvenni: ALL.